test/NuGet.Clients.Tests/NuGet.CommandLine.Test/MSBuildUtilityTest.cs

```diff
@@ -21,7 +21,7 @@ public void HighestVersionSelectedIfMsBuildVersionIsNull(List<MsBuildToolset> to
                 userVersion: null,
                 console: null,
                 installedToolsets: toolsets.OrderByDescending(t => t),
-                getMsBuildPathInPathVar: () => null);
+                getMsBuildPathInPathVar: () => null).Path;
```

Comment:
Not sure, but do we have tests which are running with MSBuild 14.0 as well as 15.0 and checking if we are correctly setting `SkipNonExistentTargets` and `BuildInParallel`?

Reply:
We don't have the infrastructure for that whatsoever. 
Adding any sort of tests for this would take quite some time. Some like having msbuild 14.0 and 15.0 on the same machine are probably very difficult to achieve in the current infrastructure.

I can create a future task for it, but I don't think it's a viable thing for 15.8. 

Thoughts?